lib/beyonic/collection.rb

```diff
@@ -2,4 +2,10 @@
 class Beyonic::Collection < OpenStruct
   include Beyonic::AbstractApi
   set_endpoint_resource "collections"
-end
\ No newline at end of file
+
+  def self.claim(amount, payload={})
```

Comment:
Claim also requires the phonenumber and remote_transaction_id, so this should also include phonenumber and remote_transaction_id, I think.